src/PreviewButton.php

```diff
@@ -99,29 +92,29 @@ public function init()
 
                 if ($versionId) {
                     $previewLink .= '?' . $this->getSettings()->versionParameter . '=' . $versionId;
-                    $text = 'Preview this version';
+                    $previewText = 'Preview this version';
                 } else {
                     $previewLink .= '?' . $this->getSettings()->draftParameter . '=' . $draftId;
-                    $text = 'Preview this draft';
+                    $previewText = 'Preview this draft';
                 }
-
-                $html = '
-                <div class="meta">
-                    <div class="field">
-                        <div class="heading">
-                            <label>Preview</label>
-                        </div>
-                        <div class="input ltr">
-                            <a href="' . $previewLink . '" class="btn">' . $text . '</a>
-                        </div>
-                    </div>
-                </div>';
+            } else {
+                $previewLink = $entry->url;
+                $previewText = 'View live page';
             }
 
-            return $html;
+            return "
+            <div class='meta'>
+                <div class='field'>
+                    <div class='heading'>
+                        <label>Preview</label>
```

Comment:
Wondering if we can give this a better title in the case where it's a link to the live version?

Reply:
Done